typescript-template-literals/main.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+const number = 9200;
+const street = 'Irvine Center Drive';
+const address = `${number} ${street}`;
+console.log('LearningFuze:', address);
+const firstName = 'Uzair';
+const lastName = 'Ashraf';
+const age = 25;
+const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+console.log('Bio:', bio);
+const fruit = 'apple';
+const item = 'pen';
+const applePen = fruit + item;
```

Comment:
This should be converted to a template literal. There is nothing wrong with this as it is. The exercise is just asking for any concatenation to be converted to template literals.